### DISCUSSION TITLE
nginx error 80

### DISCUSSION THREAD

**Discussion by ChoeruSukma** [leokhoa/laragon#839] [General]
![image](https://github.com/user-attachments/assets/c31e59ca-da75-4ad5-bd81-5606122e8b72)

**Comment by ChoeruSukma** [comment]
![image](https://github.com/user-attachments/assets/55f3b031-6b58-4fda-a412-d9ffc9c5340a)

**Comment by rescenic** [comment]
cmd as admin:
net stop winnat
Start All Laragon Services
net start winnat
